python/ray/_private/log_monitor.py

```diff
@@ -194,27 +211,31 @@ def _close_all_files(self):
 
     def update_log_filenames(self):
         """Update the list of log files to monitor."""
+        monitor_log_paths = []
         # output of user code is written here
-        log_file_paths = glob.glob(f"{self.logs_dir}/worker*[.out|.err]") + glob.glob(
-            f"{self.logs_dir}/java-worker*.log"
-        )
+        monitor_log_paths += glob.glob(
+            f"{self.logs_dir}/worker*[.out|.err]"
+        ) + glob.glob(f"{self.logs_dir}/java-worker*.log")
         # segfaults and other serious errors are logged here
-        raylet_err_paths = glob.glob(f"{self.logs_dir}/raylet*.err")
+        monitor_log_paths += glob.glob(f"{self.logs_dir}/raylet*.err")
         # monitor logs are needed to report autoscaler events
-        monitor_log_paths = glob.glob(f"{self.logs_dir}/monitor.log")
+        # TODO(rickyx): remove this after migration.
+        if not self.is_autoscaler_v2:
+            # We publish monitor logs in autoscaler v1
+            monitor_log_paths += glob.glob(f"{self.logs_dir}/monitor.log")
```

Comment:
Q: Should we just print event_AUTOSCALER.log for both v1 & v2? Isn't v1 also writing all events to this file?